themes/default/content/docs/guides/pulumi-packages/how-to-author.md

```diff
@@ -116,7 +116,21 @@ managers (npm, NuGet, Pip, GitHub) host the SDKs, but we need to know where the
 `pulumi plugin install ${NAME} ${VERSION} --server ${pluginDownloadURL}`. If `pluginDownloadURL` is not supplied, then the Pulumi
 CLI assumes the plugin is hosted at `get.pulumi.com`.
 
-### Publish the documentation
+### GitHub
```

Comment:
can we do a more descriptive header here?

Reply:
"Support for github releases" ?

Reply:
sounds good!